src/rules/declaration-property-value-no-unknown/index.js

```diff
@@ -0,0 +1,287 @@
+// NOTICE: This file is generated by Rollup. To modify it,
+// please instead edit the ESM counterpart and rebuild with Rollup (npm run build).
+"use strict";
+
+const cssTree = require("css-tree");
+const isPlainObject = require("is-plain-object");
```

Comment:
These new dependencies should be added to the `package.json` as `dependencies` to avoid things breaking in case Stylelint removes them from their dependencies.